src/core/data/flexible_type/flexible_type.hpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright © 2017 Apple Inc. All rights reserved.
+    /* Copyright © 2017 Apple Inc. All rights reserved.
```

Comment:
probably don't want this space.

Reply:
> Looks like you have changes here from #2166. That's probably not what you want.

That is what I want; gitlab doesn't seem to pass without #2166, so this branch is based off of that so I can get it to pass.  Once #2166 gets merged into master, it will drop from this PR.  Yes, dependent PRs are annoying, but it's the best I can do here.